README.md

```diff
@@ -176,13 +176,13 @@ Jenkins can use to build this. For example, there is a
 #### Update a container
 
 Once you have changed the Dockerfile for a Jenkins node, build the image and
-push it to ECR by going to the directory for the node (e.g. docker/sbt). Log
-into ECR as above, then run:
+push it to ECR.
 
-  ```
-  docker build -t nationalarchives/jenkins-build-<name-of-node>:latest .
+```
+  cd docker/nodes
+  docker build -f <name-of-node>/Dockerfile -t nationalarchives/jenkins-build-<name-of-node>:latest .
```

Comment:
Sorry, just noticed - are these the old instructions for Docker Hub rather than ECR?

Reply:
They are yeah, I've updated them to work with ECR.